symplyphysics/core/geometry/line.py

```diff
@@ -0,0 +1,20 @@
+from sympy import symbols, Point, Line, Expr, solve
+
+def two_point_function(p1: Point, p2: Point, x: Expr) -> Expr:
```

Comment:
You can put points in tuple.

Reply:
You mean to have `tuple[Expr, Expr]` as the type of `p1` and `p2` or have a union of that and `Point2D`?

Reply:
`tuple[Point, Point]`

Reply:
How is that better than having two parameters?

Reply:
You won't have point1 and point2. Just some set of points.

Reply:
I think it's better to restrict it to two parameters. It would make sense to have a tuple if we were to allow any arbitrary collection of points of any length. But we specifically construct the equation using two points. If we have a tuple of points on hand, say `points`, we can always pass `*points` as the argument.

Reply:
Tuple is not of arbitrary length. Tuple[Point, Point] is for 2 points. Usually having variables with names like `var1`, `var2` is a red flag. But it is not a big deal so choose yourself if you want to pack them in a tuple or not.

Reply:
All of `sympy` uses `p1` and `p2` arguments, at least up to the point I've seen, so I wanted it to be similar to that.

Reply:
Ok

Reply:
It's done then

Reply:
But other issues are not fixed.

Reply:
Ah, forgot to sync